cryptofeed/standards.py

```diff
@@ -42,8 +44,8 @@ def load_exchange_pair_mapping(exchange: str):
             _std_trading_pairs[std] = {exchange: exch}
 
 
-def get_exchange_info(exchange: str):
-    mapping = gen_pairs(exchange)
+def get_exchange_info(exchange: str, key_id: str):
```

Comment:
why did this change?

Reply:
WHALE_ALERT feed required an API key even to query recognized coins.
So I need to forward the API key to all functions that call `gen_pairs()`.
This is the case for `get_exchange_info`.

> a few other things - I think the name of the yaml file with the keys should be different, maybe just call it `config.yaml` so it can be used for other things. Also the config should be at the feedhandler level, not a feed level. feedhandler can load the config and various parts of the system can use it as applicable.


Bryant,
The sole purpose of `feed_keys.yaml` is to provide to the feed requiring it (at the moment only WHALE_ALERT) a `key` class parameter without requiring creation of an object.
The sole need is to be able to call the class method `info` without more parameter.
If we move this config file at `FeedHanlder` level, then this need is not satisfied. `key` cannot be made a class parameter of `Feed` class.

In this case, the other approach to have `info` class method working (that I just realize now) is simply to add a `key_id` parameter to that function:
```python
@classmethod
def info(cls, key_id: str = None) -> dict:
```

I can do that, but then I don't need any longer a config file.
So, if you agree, if you feel that a config file for `Feed` class is not appropriate (and I have no trouble with that, no worries), I will simply remove it, and do as proposed above: add a `key_id` parameter to `info` class method.

Would this be ok?
(adding a config file at `FeedHandler` level could then be managed in a separate PR, as a separate feature)

Just let me know what is you preference, and I will adjust accordingly.
Bests,